block.go

```diff
@@ -272,11 +284,20 @@ func OpenBlock(dir string, pool chunkenc.Pool) (*Block, error) {
 		return nil, err
 	}
 
-	tr, err := readTombstones(dir)
+	tr, tsr, err := readTombstones(dir)
 	if err != nil {
 		return nil, err
 	}
 
+	// TODO refactor to set this at block creation time as
```

Comment:
Will this be added in this PR?

Reply:
yes, I couldn't find a good way to calculate the block size at its creation so would like to revisit this at some point.